nvme-print.c

```diff
@@ -1773,17 +1773,23 @@ static void nvme_show_id_ctrl_cmic(__u8 cmic)
 static void nvme_show_id_ctrl_oaes(__le32 ctrl_oaes)
 {
 	__u32 oaes = le32_to_cpu(ctrl_oaes);
-	__u32 rsvd0 = (oaes & 0xFFFF8000) >> 15;
+	__u32 rsvd0 = (oaes & 0xF0000000) >> 28;
+	__u32 zicn = (oaes & 0x08000000) >> 27;
+	__u32 rsvd1 = (oaes & 0x07FF8000) >> 15;
 	__u32 nace = (oaes & 0x100) >> 8;
 	__u32 fan = (oaes & 0x200) >> 9;
 	__u32 anacn = (oaes & 0x800) >> 11;
 	__u32 plealcn = (oaes & 0x1000) >> 12;
 	__u32 lbasin = (oaes & 0x2000) >> 13;
 	__u32 egealpcn = (oaes & 0x4000) >> 14;
-	__u32 rsvd1 = oaes & 0xFF;
+	__u32 rsvd2 = oaes & 0xFF;
 
 	if (rsvd0)
-		printf(" [31:10] : %#x\tReserved\n", rsvd0);
+		printf(" [31:28] : %#x\tReserved\n", rsvd0);
+	printf("[27:27] : %#x\tZone Information Change Notices %sSupported\n",
```

Comment:
Sorry for finding this late.

Isn't this field called "Zone Descriptor Changed"?

Should it be "Zone Descriptor Changed Event %sSupported"?

Reply:
for this 
LBA Status Information Notices event(spec) -> LBA Status Information Notices %sSupported\n"(nvme-cli)
Zone Descriptor Changed Notices(spec) -> Zone Information Change Notices %sSupported\n" (nvme-cli)

Reply:
LBA Status Information Notices has "Information" in it, that is not true to Zone Descriptor Changed Notices.

I believe it should be "Zone Descriptor Changed Notices %sSupported", else an unknown person would try to lookup Zone Information Changed Notices in the spec, and come up empty-handed.